src/block/EndPortalFrame.php

```diff
@@ -58,4 +63,16 @@ public function getLightLevel() : int{
 	protected function recalculateCollisionBoxes() : array{
 		return [AxisAlignedBB::one()->trim(Facing::UP, 3 / 16)];
 	}
+
+	public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player $player = null, array &$returnedItems = []) : bool{
+		if(!$item->equals(VanillaItems::ENDER_EYE()) || $this->eye){
```

Comment:
Less expensive check should be placed first. Also, this should use `ItemTypeIds`.